handwriting_recognition.ipynb

```diff
@@ -0,0 +1,1027 @@
+{
```

Comment:
<ol><li>Aspect ratio is preserved</li><li>Content of the image isn't distorted</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sayakpaul/Handwriting-Recognizer-in-Keras/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>